quantized_mesh_encoder/normals.py

```diff
@@ -12,7 +12,7 @@ def compute_vertex_normals(positions, indices):
     # positions and indices are both arrays of shape (-1, 3)
     # `coords` is then an array of shape (-1, 3, 3) where each block of (i, 3,
     # 3) represents all the coordinates of a single triangle
-    tri_coords = positions[indices]
+    tri_coords = positions[indices].astype('float32')
```

Comment:
Why do you need this? Isn't `positions` already always `float32`? 
https://github.com/kylebarron/quantized-mesh-encoder/blob/b20cb9a9f227b2a066930ff5e1ca367638e8b5ed/quantized_mesh_encoder/encode.py#L57

Reply:
I too was a bit confused why it was erroring out without the above but now I see why...

I'm passing it the wrong `positions` from the `encode()` function --- mistakenly passed the result of  interp_positions. I think in actuality it should probably be getting the positions in ecef. Will rework this pull request with this and the other issues you've pointed out later this week